apidoc/Polyline.yml

```diff
@@ -31,6 +31,11 @@ properties:
     summary: Line width in pixels to use when drawing the polyline.
     type: Number
     default: 10
+    
+  - name: touchEnabled
+    summary: Determines whether view should receive touch events.
+    type: Boolean
+    default: true
```

Comment:
would be good to have that too. If you create a polyring (path around an area) with annoations in the middle you'll get weird results clicking on an annotation. E.g. you can't deselect a selected annotation by clicking inside the ring on the map